lisa/sut_orchestrator/azure/features.py

```diff
@@ -1517,13 +1517,12 @@ def remove_data_disk(self, names: Optional[List[str]] = None) -> None:
     def get_resource_disk_mount_point(self) -> str:
         # get customize mount point from cloud-init configuration file from /etc/cloud/
         # if not found, use default mount point /mnt for cloud-init
-        if self._node.shell.exists(
-            self._node.get_pure_path("/var/log/cloud-init.log")
-        ) and self._node.shell.exists(
-            self._node.get_pure_path("/var/lib/cloud/instance")
-        ):
-            self._log.debug("Disk handled by cloud-init.")
+        ls_tool = self._node.tools[Ls]
```

Comment:
It should be a shared method in Azure, `is_cloud_init_enabled`.

Reply:
updated